sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -127,7 +121,7 @@ def _get_url_for_timerange(self, timerange, **kwargs):
             return SRSClient._get_default_uri()
         result = list()
         base_url = 'ftp://ftp.swpc.noaa.gov/pub/warehouse/'
-        total_days = int(timerange.days) + 1
+        total_days = int(np.round(timerange.days.value) + 1)
```

Comment:
I am not sure this is what I want to do. 

 In the example that triggered this, the `days.value` was `0.9`, so `int(0.9) + 1` is one day. But this has made it `round + 1`.